CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## [v6.7.1] (???)
 
+## Added
+
+* Added new `completions` package for dynamically generating descriptive completions
```

Comment:
So far, we've only used the changelog for changes that are visible to the user. I'm not sure in this case if we should add this, however I have no problem with keeping it in though